tools/etc/rsyslog.d/45-frr.conf

```diff
@@ -1,6 +1,12 @@
 # The lines below cause all FRR daemons and process to go
 # to /var/log/frr/frr.log, then drops the message so it does
 # not also go to /var/log/syslog, so the messages are not duplicated
+# Set file ownership of the frr.log file so frr can reference it from
+# CLI. Without this, log file is created with root ownership which can
+# cause issues.
+
+$FileOwner frr
+$FileGroup frrvty
```

Comment:
In addition to the intrusion problem described in my conversation comment, I'm not convinced `frrvty` is the right group here.  It's the group that controls vty access, which human operator users can be added to so they can use `vtysh` without being root.  If we want to give them read access, then this makes sense, but then we also need to make sure the permissions are 0640.  If the permissions include group write, this opens a hole that suddenly allows operators to tamper with the logs.  If the permissions are 0644, the group doesn't matter.

So either this is accompanied by a permission setting to 0640, or the group setting is underdetermined in its effect and needs to go.